docs/man/git-lfs-checkout.1.ronn

```diff
@@ -4,14 +4,16 @@ git-lfs-checkout(1) -- Update working copy with file content if available
 ## SYNOPSIS
 
 `git lfs checkout` <filespec>...
+`git lfs checkout` --to <path> { --ours | --theirs | --base } <file>...
 
 ## DESCRIPTION
 
-This command is deprecated, and should be replaced with `git checkout`.
+This command is deprecated when used without `--to`, and should be replaced with
```

Comment:
I wonder if it makes sense to continue to mark this command as deprecated in this case? I'm also not immediately sure what the behavior of `git checkout` is when given a `--{ours,theirs,base}` flag, and whether or not upstream runs the process filter.

Reply:
Yeah, I wasn't really sure how to go about that deprecation notice.  Removing it is fine with me.